src/r/README.Rmd

```diff
@@ -0,0 +1,48 @@
+---
+output: github_document
+---
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "man/figures/README-",
+  out.width = "100%"
+)
+```
+
+# psychro
```

Comment:
Why not psychrolib?

Reply:
At the beginning of this PR, I suggested to rename the R package from psychrolib to psychro, because on CRAN, 'lib' is not commonly used in package names. But later I changed it back to psychrolib for consistency. I somehow forgot to change the name in README. Will correct it.

Reply:
By the way, right now without committing the `man` folder, users will not get documentation via `remotes::install_github`. So I am thinking to add more instructions similar like those in `DEVELOP.md`

Also, as you removed `README.md` which is created via compiling `README.Rmd`, the user will not be able to see any rendered README when browsing to `src/r`. So that's why I get a little bit confused by the purpose of **Deployment** section in `DEVELOP.md`:

* If it is for users who just want to install the development version of the R package, it is better to have a separated rendered README in `src/r`, instead of `DEVELOP.md`, since they are not interested in development. However, as we did not have a rendered README.

Reply:
`README.md` is generated from  `README.Rmd` -- I don;t see any reason for included the README artifacts in the repo. End-users are not supposed to go to `src/r` but to navigate from the main readme to the CRAN repo where the correct documentation is displayed. Does this make sense?

Reply:
Yes, it makes sense. In this case, the `README.Rmd` will only needed to be compiled for CRAN submission, as README.md will be linked in `Materials` field on the package CRAN page. If so, I don't think we need `README.Rmd` anymore. We can just make sure the main README and related figures are copied into `src/r` before CRAN submission. And hyperlinks are correctly updated.
Will write an R script to do this, together with all CRAN submission preparation stuff, and put it under `src/r/tools/`.

Reply:
If we use the main README, I see there is a link for only online API docs for python, which might not be very useful for R users.

Is it possible to host 2 separate docs on GitHub pages? With current setup, it is easy to create an R package site using [pkgdown](https://pkgdown.r-lib.org/), similar like Sphinx. Examples are the pkgdown website itself and my package [eplusr](https://hongyuanjia.github.io/eplusr/)

Reply:
I see your point. We have a note that reads "Please note that although the API describes the Python version of the library, the API is common across all the supported language implementations." in the current README. Where possible I would prefer to keep this as simple as possible. How many factions have had their signature changed or do not exist any more? If this does not apply we would could include a specific link to R but I would prefer to have only a reference page rather than a full website for R (i.e. only something like https://pkgdown.r-lib.org/reference/index.html). @didierthevenard what is your opinion on this as well?

Reply:
All function names are the same. There are 2 main differences compared to the python APIs:

1. Constants, like `ZERO_FAHRENHEIT_AS_RANKINE` are not exported, i.e. not directly accessible to users.
2. For calculations, all functions in R accept a vector, not a scalar. Bulk calculations, like `CalcPsychrometricsFromRelHum` returns an R list.

Reply:
In this case, I would probably just expand the note we have already and point to the python API. What do you think? But let's also wait for @didierthevenard comments.

Reply:
I think that will do, if we want to keep it as simple as possible. Let's also wait for @didierthevenard thoughts.

Reply:
I agree with expanding the note and have it point to the Python API.

Reply:
@hongyuanjia are you OK to expand this in the README?

Reply:
@dmey Sure.

Reply:
no problems  -- had a chance do do it now -- done in https://github.com/psychrometrics/psychrolib/pull/49/commits/1af14d0a7fc9abb78a44d46ca01ab6ef7f8a6e17